crates/node-bindings/src/anvil.rs

```diff
@@ -20,14 +20,24 @@ const ANVIL_STARTUP_TIMEOUT_MILLIS: u64 = 10_000;
 /// Construct this using [`Anvil`].
 #[derive(Debug)]
 pub struct AnvilInstance {
-    pid: Child,
+    child: Child,
     private_keys: Vec<K256SecretKey>,
     addresses: Vec<Address>,
     port: u16,
     chain_id: Option<u64>,
 }
 
 impl AnvilInstance {
+    /// Returns a reference to the child process.
+    pub fn child(&self) -> &Child {
```

Comment:
can we spin these changes out into a separate pr?

Reply:
Will do later